Tests/Indicators/AlphaTests.cs

```diff
@@ -0,0 +1,75 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using NUnit.Framework;
+using QuantConnect.Data.Consolidators;
+using QuantConnect.Data.Market;
+using QuantConnect.Indicators;
+using System;
+using static QuantConnect.Tests.Indicators.TestHelper;
+
+namespace QuantConnect.Tests.Indicators
+{
+    [TestFixture]
+    public class AlphaTests : CommonIndicatorTests<TradeBar>
+    {
+        protected override string TestFileName => "alpha_datatest.csv";
+
+        protected override string TestColumnName => "Alpha";
+
+        private DateTime _reference = new DateTime(2020, 1, 1);
+
+        protected override IndicatorBase<TradeBar> CreateIndicator()
+        {
+            var indicator = new Alpha("testAlpha", "SPY_", "SPY", 5);
```

Comment:
Tests are not working. Since the indicator has input for two symbols, the test dataset file must be properly formated for this.

- You need a "Symbol" column for the symbol value the row data is for
- A "Close" column with the close price for the symbol
- See the Beta indicator test data file (bi_datset.csv) and the `BetaIndicatorTests` class to see how the symbols are passed in the indicator constructor.
- Make sure to run all common tests inherited from `CommonIndicatorTests<T>` class.